curriculum/challenges/english/10-coding-interview-prep/rosetta-code/straddling-checkerboard.md

```diff
@@ -8,7 +8,11 @@ dashedName: straddling-checkerboard
 
 # --description--
 
-Implement functions to encrypt and decrypt a message using the [straddling checkerboard](https://en.wikipedia.org/wiki/Straddling_checkerboard) method. The functions will take a string and an array as parameters. The array has 3 strings representing the 3 rows of the checkerboard. The output will be a series of decimal digits. Numbers should be encrypted by inserting the escape character before each digit, then including the digit unencrypted. This should be reversed for decryption.
+Implement functions to encrypt and decrypt a message using the **straddling checkerboard** method.
+
+Each function will take a two arguments, a `message` string and  the `alphabet` array. The `alphabet` contains 3 strings representing the 3 rows of the checkerboard. 
+
+The output of the `straddle()` function is a series of decimal digits. Numbers should be encrypted by inserting the escape character before each digit, then including the digit <em>unencrypted</em>. This should be reversed for decryption with the `unstraddle()` function.
```

Comment:
```suggestion
The output of the `straddle()` function should be a series of decimal digits. Numbers should be encrypted by inserting the escape character before each digit, then including the digit <em>unencrypted</em>.

This should be reversed for decryption with the `unstraddle()` function.
```

Reply:
I do like the suggestions, I am implementing them